src/vma/util/verbs_extra.cpp

```diff
@@ -314,3 +314,14 @@ int priv_ibv_query_flow_tag_supported(struct ibv_qp *qp, uint8_t port_num)
 	return res;
 }
 
+
+int vma_rdma_lib_reset() {
+#ifdef DEFINED_RDMA_LIB_RESET
+	vlog_printf(VLOG_DEBUG, "rdma_lib_reset called");
+	return rdma_lib_reset();
```

Comment:
@miked, according to its manpage the AC_CHECK_LIB will only add the -lrdmacm if it is found and there is no other action specified.
This differs from AC_SEARCH_LIBS which adds it regardless if there are other actions specified.
The fact is that there are no build issues when the lib is found by this check.

Re. The %m, the manpage states that its argument is ignored (maybe the rest of the arguments - same thing in this case) is ignored. I didn't request to change this so as to remain in convention with how its done throughout the file.